hydroshare/hs_elements.py

```diff
@@ -502,7 +503,11 @@ def resource_creation_type(self, index):
             '#dropdown-resource-type ul li:nth-of-type({})'.format(index),
         )
 
+class DashboardPage:
+    def __init__(self):
+        self.get_started_toggle = SiteElement(By.ID, 'change_me')
```

Comment:
The 'change_me' ID works, but I'd recommend somehow noting this as a TODO in hydroshare - using a non-placeholder ID.  Something like id="get_started_toggle". If you can change that in the source code quickly, then we can change the test before merging the PR.  Whatever you think is best.

Reply:
yes. I will plan to once the dashboard pr is approved.

Reply:
updated id="id-getting-started-toggle" 
When beta.hydroshare.org is online, you will see the new id.